src/Clockify/Fill/EntryFillDialog.cs

```diff
@@ -247,12 +251,13 @@ public class EntryFillDialog : ComponentDialog
             }
         }
 
-        private async Task<DialogTurnResult> AddEntryAndExit(DialogContext stepContext,
+        private async Task<DialogTurnResult> AddEntryAndExit(WaterfallStepContext stepContext,
```

Comment:
Why this change?

Reply:
I needed to access the Values collection to get the TimeZone